live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommand.java

```diff
@@ -42,7 +42,8 @@ public record IngestMetadataCommand(
         PropertyDelta<String> modifiedByUserWithId,
         PropertyDelta<Set<String>> aspectNames,
         PropertyDelta<Long> createdAt,
-        Set<CustomPropertyDelta<?>> properties)
+        Set<CustomPropertyDelta<?>> properties,
+        boolean contentRemoved)
```

Comment:
Maybe instead we could have `List<String> contentPropertiesRemoved` here?

Reply:
Or maybe even better we could represent it internally as `CustomPropertyDeleted` in the `Set<CustomPropertyDelta<?>> properties` field?

Reply:
I like this idea much more, but I would still create separate command for that - it costs us basically nothing and will reduce the complexity

Reply:
we agreed to put content deletion info in `Set<CustomPropertyDelta<?>> properties`